### PR TITLE
Add Open Liberty stack devfile that uses Gradle

### PR DESCRIPTION
**Pull request by mezarin**
FYI @scottkurz, @elsony

### CODE REVIEW

**Review by scottkurz — APPROVED** [review]
LGTM

File: stacks/java-openliberty-gradle/devfile.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+schemaVersion: 2.0.0
+metadata:
+  name: java-openliberty-gradle
+  version: 0.1.0
+  displayName: "Open Liberty Gradle"
+  description: Java application Gradle built stack using the Open Liberty runtime
```
Comment (johnmcollier) [inline stacks/java-openliberty-gradle/devfile.yaml:22]: We're trying to make sure all of the stacks have icons associated with them. Could you add an icon for this stack? This is the one we're using for the other openliberty stack: https://github.com/devfile/registry/blob/main/stacks/java-openliberty/devfile.yaml#L23

**Review by johnmcollier — CHANGES_REQUESTED** [review]
Generally looks good to me and the tests are passing 👍

One comment regarding icons, otherwise I have no concerns. CC @elsony

**Review by johnmcollier — APPROVED** [review]
LGTM. Thanks!

File: stacks/java-openliberty-gradle/devfile.yaml (hunk 1 above)
Comment (mezarin) [reply stacks/java-openliberty-gradle/devfile.yaml:22]: @johnmcollier Thanks for taking a look. I added the icon entry.

**Comment by mezarin** [comment]
Hi @elsony, @johnmcollier, just checking back to see if there is anything else needed on my part for this PR to be merged. Thanks.

**Pull request merged**